PowerEditor/installer/nativeLang/portuguese.xml

```diff
@@ -158,11 +158,11 @@
                     <Item id="43021" name="Remover Linhas Marcadas"/>
 					<Item id="43051" name="Remover Linhas Desmarcadas"/>
 					<Item id="43050" name="Inverter Marcador"/>
-					<Item id="43052" name="Encontrar carateres ..."/>
+					<Item id="43052" name="Encontrar carateres..."/>
```

Comment:
hey,
I am not a programmer or any kind of work-tree related, I am a simple 
user, that likes notepad ++ and wanted a better translation. just that. 
before I edit the translation there were some missing terms, bad 
translations, etc.
I removed all the &amp; things becouse I dont think it needs it. as I 
said, I am not a programmer, and I dont even know the language used to 
write the lang.xml, but I dont think it needs the term &amp;, because I 
am using my edited xml file, and the basic things like menus, submenus, 
is working fine without the term &amp;, didn't tryed something more 
advanced yet.

Is it really necessary? I dont see the point of using the &amp; since 
without it is working well and besides, at notepad++ website they say 
that is something to split text or something, dont really remember, on 
the website the author dont says anything about &amp; being a shortcut 
and as third point, why spliting the text with the term? people can't 
put it on the end or the beginning of the line? why: h&amp;ome instead 
of home&amp; or &amp;home??? its way more complicated to read the 
text... I made a mistake at the first lang.xml file becouse of that.
-- 
João Pereira